easybuild/tools/github.py

```diff
@@ -627,6 +627,22 @@ def _easyconfigs_pr_common(paths, start_branch=None, pr_branch=None, target_acco
 
     _log.debug("git status: %s", git_repo.git.status())
 
+    # seperate easyconfigs and patch files
+    ec_paths = [p for p in existing_paths if p.endswith('.eb') or p.endswith('.yeb') or not is_patch_file(p)]
```

Comment:
the two `endswith` checks are useless if you also check `not is_patch_file`, since non-easyconfig files will fall through on the last check anyway

so, just use `not is_patch_file(p))` (and assume only easyconfigs are passed in)